fabfile/run.py

```diff
@@ -17,3 +23,82 @@ def regression():
     'Runs Citus\' regression tests'
     with cd(config.paths['citus-repo']):
         run('make check')
+
+@task
+@runs_once
+@roles('master')
+def dml_tests(filename):
+    'Runs pg-bench dml tests using the given config options'
+    dir_path = '/home/ec2-user/'
+    results_file = open(dir_path + 'dml_benchmark_results.csv', 'w')
+    config_file = open(filename)
+
+    insert_line = config_file.readline()
+    insert_configs = dict(item.split(":") for item in insert_line.split())
+
+    update_line = config_file.readline()
+    update_configs = dict(item.split(":") for item in update_line.split())
```

Comment:
I'd have a slight preference over creating a `DML_Benchmark` (or such) object with all the parameters provided in the file. Later, read the file and generate the corresponding objects (i.e., one object per `benchmark, shard_count, replication_factor, postgres_version, citus_version` ) and run them one by one. 

That seems to hide a lot of details in this loop and make it a lot more readable. Plus, that'd make expanding the tests simpler.